extractor/src/main/java/org/schabi/newpipe/extractor/utils/Parser.java

```diff
@@ -124,9 +125,7 @@ public static String[] getLinksFromString(final String txt) throws ParsingExcept
                 links.add(txt.substring(ls.getBeginIndex(), ls.getEndIndex()));
             }
 
-            String[] linksarray = new String[links.size()];
-            linksarray = links.toArray(linksarray);
-            return linksarray;
```

Comment:
I think this method is just unused... Should we remove it or keep it in case it becomes handy later? In the second case, just change the function signature and avoid converting `links` to an array